usaspending_api/etl/tests/integration/test_load_to_from_delta_subawards.py

```diff
@@ -0,0 +1,785 @@
+from datetime import datetime, date
+from decimal import Decimal
+
+from pytest import mark
+
+from usaspending_api.etl.tests.integration.test_load_to_from_delta import (
+    create_and_load_all_delta_tables,
+    verify_delta_table_loaded_to_delta,
+)
+
+
+@mark.django_db(transaction=True)
+def test_load_table_to_delta_for_subawards(
+    spark,
+    s3_unittest_data_bucket,
+    broker_server_dblink_setup,
+    populate_broker_data_to_delta,
+    populate_data_for_transaction_search,
+):
```

Comment:
I think this test needs the `hive_unittest_metastore_db` fixture on it, so that it keeps track of delta DBs/tables created and destroyed during the course of a test session. It may just be benefitting from one of the other tests having this session-fixture running before it, but it should have it.